test/Tests/src/Semantic/Conversion_Spec.enso

```diff
@@ -115,4 +116,11 @@ spec =
         Test.specify "should not allow currying" <|
             Panic.recover Any (Foo.from) . catch Any .to_display_text . should_equal "Conversion currying without `that` argument is not supported."
 
+        Test.specify "Use conversions to in Conversion_Use module" <|
+            Hello.formulate [ Hello.Say "Proper", Hello.Say "Type" ] . should_equal "ProperType"
+            Hello.formulate [ Foo.Value "Perform", Bar.Value "Conversion" ] . should_equal "PERFORM conversion!"
```

Comment:
This line invokes the static `formulate` method with non-`Hello` instances, but these instances have conversions methods to `Hello` defined in this module. See below...